src/Homework4/BrandNewGreenhouse.java

```diff
@@ -1,25 +1,183 @@
 package Homework4;
 
+import jdk.jshell.spi.ExecutionControl;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.InputMismatchException;
 import java.util.List;
 import java.util.ArrayList;
+import java.util.Scanner;
 
-public class BrandNewGreenhouse implements Greenhouse {
+public class BrandNewGreenhouse implements Greenhouse, Illuminated {
+    private static Scanner scanner = new Scanner(System.in);
     private List<Plant> plants;
     private double temperature;
+    private File file;
+
     public BrandNewGreenhouse() {
         plants = new ArrayList<>();
+        file = new File(".\\src\\Homework4\\MyGreenhouse.txt");
     }
-    public static void main( String[] args ) {
+
+    @Override
+    public void illuminateRed() {
+        try {
```

Comment:
You don't have to use try catch here, you can just add to the method public void illuminateRed() throws some_exception. 
For example https://www.javatpoint.com/throws-keyword-and-difference-between-throw-and-throws